packages/headless/src/features/facets/facet-set/interfaces/response.ts

```diff
@@ -5,3 +5,8 @@ export interface FacetValue extends BaseFacetValue {
 }
 
 export type FacetResponse = BaseFacetResponse<FacetValue>;
+
+export type AutomaticFacetResponse = Omit<
+  BaseFacetResponse<FacetValue>,
+  'facetId'
```

Comment:
The facets returned by the automatic feature follow the same pattern as `BaseFacetResponse<FacetValue>` except they don't have a `facetId`.